components/camera/videosource/static.go

```diff
@@ -72,8 +72,8 @@ type fileSource struct {
 // fileSourceAttrs is the attribute struct for fileSource.
 type fileSourceAttrs struct {
 	*camera.AttrConfig
-	Color string `json:"color"`
-	Depth string `json:"depth"`
+	Color string `json:"color_url"`
+	Depth string `json:"depth_url"`
```

Comment:
these are not URLs, but file paths, so it should be something like "color_file_path" and "depth_file_path"